src/main/scala-2.12-/com/fulcrumgenomics/commons/Compat.scala

```diff
@@ -0,0 +1,12 @@
+package com.fulcrumgenomics.commons
+
+import scala.annotation.ClassfileAnnotation
+
+/** Trait that includes types used for cross-building compatibility; scala 2.13 version. */
+private[commons] trait Compat {
+  /** For annotations available at runtime in 2.13 we use ConstantAnnotation. */
+  type ConstantAnnotation = scala.annotation.ClassfileAnnotation
+
+  /** For basic views we use the new scala.collection.View. */
+  type View[+A] = scala.collection.IterableView[A, Iterable[A]]
```

Comment:
This isn't strictly needed for commons, but we need it in fgbio, so I think defining it here is a good idea.